docs/guide/deployment/resource-aggregate.md

```diff
@@ -1,74 +1,152 @@
 # Resource aggregate
 
-## Description
```

Comment:
Next time please split it to multiple MRs.
I will review it after resolving all comments above.